sdk/synapse/Azure.Analytics.Synapse.Spark/tests/SparkTestUtilities.cs

```diff
@@ -1,32 +1,34 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
 using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
 using Azure.Analytics.Synapse.Spark.Models;
 using Azure.Analytics.Synapse.Tests;
+using Azure.Core.TestFramework;
+using Azure.Analytics.Synapse.Spark;
 
-namespace Azure.Analytics.Synapse.Spark.Tests
+namespace Azure.Analytics.Synapse.Tests
 {
-    internal static class SparkTestUtilities
+    public static class SparkTestUtilities
     {
         /// <summary>
         /// Create parameters for Spark batch tests.
         /// </summary>
         /// <param name="test"></param>
         /// <returns></returns>
-        internal static SparkBatchJobOptions CreateSparkJobRequestParameters(this SparkClientTestBase test)
+        public static SparkBatchJobOptions CreateSparkJobRequestParameters(TestRecording recording, SynapseTestEnvironment testEnvironment)
         {
-            string name = test.Recording.GenerateName("dontnetbatch");
-            string file = string.Format("abfss://{0}@{1}.dfs.core.windows.net/samples/java/wordcount/wordcount.jar", test.TestEnvironment.StorageFileSystemName, test.TestEnvironment.StorageAccountName);
+            string name = recording.GenerateName("dontnetbatch");
+            string file = string.Format("abfss://{0}@{1}.dfs.core.windows.net/samples/java/wordcount/wordcount.jar", testEnvironment.StorageFileSystemName, testEnvironment.StorageAccountName);
```

Comment:
Does it matter that these are referring to `java`.    Is this URI stable across environments/platforms/clouds?

Reply:
These tests require the uploaded [sample](https://github.com/Azure-Samples/Synapse/tree/main/Spark) from the team copied onto the shared storage, which can be either java, python, or dotnet. 

This code in question was from the client team, and defaulted to java. I'm planning on improving this in a future PR.